src/main/resources/com/google/jenkins/plugins/k8sengine/Messages.properties

```diff
@@ -6,4 +6,5 @@ KubernetesEnginePublisher.ProjectIDRequired=Project ID is required
 KubernetesEnginePublisher.NoCredential=No credential selected
 KubernetesEnginePublisher.CredentialProjectIDRequired=Project ID required to validate credential
 KubernetesEnginePublisher.CredentialAuthFailed=Credentials failed to authenticate
-KubernetesEnginePublisher.ZoneRequired=Zone is required
\ No newline at end of file
+KubernetesEnginePublisher.ZoneRequired=Zone is required
+KubernetesEnginePublisher.ZoneFillError=Error retrieving zones
```

Comment:
Looks like we were missing a trailing newline here to begin with, but would you mind adding one :)

Reply:
On the way with the new verification messages